src/components/Message/MessageText.js

```diff
@@ -58,6 +61,12 @@ const MessageTextComponent = (props) => {
     return null;
   }
 
+  const renderedText = message?.mml ? (
+    <MML source={message.mml} onSubmit={handleActionMML} />
+  ) : (
+    messageText
+  );
```

Comment:
IMO, we should stick to the attachment type, mixing up the text render with the mml could make things unnecessary complicated